registry/registry.go

```diff
@@ -130,9 +131,23 @@ func (registry *Registry) ListenAndServe() error {
 			},
 		}
 
-		tlsConf.Certificates[0], err = tls.LoadX509KeyPair(config.HTTP.TLS.Certificate, config.HTTP.TLS.Key)
-		if err != nil {
-			return err
+		if config.HTTP.TLS.LetsEncrypt.CacheFile != "" {
```

Comment:
Mutually exclusive is probably better, otherwise I think the order should probably be flipped around, Let's Encrypt should only be used if no certificate is provided.